uiplib/utils.py

```diff
@@ -17,6 +19,20 @@ def make_dir(dirpath):
     if sys.platform.startswith('linux'):
         os.chmod(dirpath, 0o777)
 
+
+def exit_UIP(service=False):
+    pid_file = os.path.join(HOME_DIR, 'daemon-uip.pid')
+    if service:
+        if os.path.exists(pid_file):
+            send(pid_file, SIGTERM)
```

Comment:
what if pid file exists and this process is already terminated? happens sometimes when UIP is unexpectadly closed in that case we need to:
1. just delete pid file and exit